native/native_main.py

```diff
@@ -153,7 +158,12 @@ def handleMessage(message):
             file.write(message["content"])
 
     elif cmd == 'temp':
-        (handle, filepath) = tempfile.mkstemp()
+        prefix = message.get('hostname')
+        if prefix is not None:
+            prefix = sanitizePrefix(prefix)
+
+        (handle, filepath) = tempfile.mkstemp(
+            prefix='moz-edit_{}_'.format(prefix))
```

Comment:
should probably default to "'tri-tmp" as it might be used for stuff other than editing